### ISSUE TITLE
Is it a Bug to compute the distillation loss?

### ISSUE THREAD

**Issue by pyun-ram** [kshmelkov/incremental_detectors#10]
I feel confused about compute_distillation_crossentropy_loss.
Let me call the main model as target_model and the model to compute the distillation loss as sub_model.
From the code, it seems the target_model compute logits with rois (https://github.com/kshmelkov/incremental_detectors/blob/master/frcnn.py#L143).
The sub_model computes distillated_logits with proposals
(https://github.com/kshmelkov/incremental_detectors/blob/master/network.py#L104)

The rois and proposals are with same shape but different contents. However, the distillation loss is computed from them in compute_distillation_crossentropy_loss().

I am not quite sure about this point. I would be grateful it if anyone can give me some hint or discussion.

**Comment by pyun-ram** [comment]
Sorry It was my bad. I figure it out you merge the rois and proposals in https://github.com/kshmelkov/incremental_detectors/blob/master/network.py#L127
